### PR TITLE
Pkg.bison

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

File: cmake/configs/default.cmake [hunk 1]
```diff
@@ -163,6 +163,7 @@ hunter_config(dbus VERSION 1.10.0-hunter-4)
 hunter_config(dest VERSION 0.8.0-p4)
 hunter_config(dlib VERSION 19.6-p0)
 hunter_config(doctest VERSION 1.2.0)
+hunter_config(double-conversion VERSION 3.0.0)
```
Comment (ruslo) [inline cmake/configs/default.cmake]: Doesn't belong here. From [README](https://github.com/ingenue/hunter):
```
The only pull requests accepting here is about continuous integration testing
on Travis/AppVeyor, i.e. should only affect .travis.yml or appveyor.yml (in some rare cases jenkins.py)
```

**Review by ruslo — CHANGES_REQUESTED** [review]

File: cmake/configs/default.cmake (hunk 1 above)
Comment (isaachier) [reply cmake/configs/default.cmake]: I think this is because of a merge. Will try reverting that merge.

File: .travis.yml [hunk 2]
```diff
@@ -51,21 +52,11 @@ matrix:
 
     # OSX {
     - os: osx
-      env: PROJECT_DIR=examples/bison TOOLCHAIN=default
```
Comment (ruslo) [inline .travis.yml]: Do not remove but comment them out. It will be easeir to enable them back.

**Review by ruslo — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (isaachier) [reply .travis.yml]: OK I think the issue here is that I merged in `pkg.template` too early. Let me just undo that for now.

File: appveyor.yml [hunk 3]
```diff
@@ -9,41 +9,9 @@ environment:
   matrix:
     # Note: 'default' toolchain is the same as 'vs-14-2015'
 
-    - TOOLCHAIN: "ninja-vs-12-2013-win64"
+    - TOOLCHAIN: "dummy"
       PROJECT_DIR: examples\bison
 
-    - TOOLCHAIN: "nmake-vs-12-2013-win64"
```
Comment (ruslo) [inline appveyor.yml:15]: Keep it:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#excluding-tests

**Review by ruslo — CHANGES_REQUESTED** [review]

**Review by ruslo — APPROVED** [review]

**Pull request merged**